packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -138,7 +148,9 @@ const UnthemedQueryBar: React.FunctionComponent<QueryBarProps> = ({
             id="query-bar-option-input-filter-label"
             className={filterLabelStyles}
           >
-            Filter
+            <Link href={queryBarDocumentationLink} target="_blank">
```

Comment:
This puts a link inside a label, need to check on the implications of that, might be something we don't want to use `label` for (then we can hidden label the text area).